Ghidra/Features/Base/data/noReturnFunctionConstraints.xml

```diff
@@ -9,6 +9,9 @@
     <functionNamesFile>ElfFunctionsThatDoNotReturn</functionNamesFile>
   </executable_format>
   <executable_format name="Mac OS X Mach-O">
+    <compiler id="golang">
+      <functionNamesFile>GolangFunctionsThatDoNotReturn</functionNamesFile>
```

Comment:
Is there a reason why `Mac OS X Mach-O` file format was ignored?

Reply:
mainly a detail that got overlooked.